server/app/views/components/Icons.java

```diff
@@ -290,6 +300,16 @@ public enum Icons {
           + " 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8zm0-14c-2.21 0-4 1.79-4"
           + " 4h2c0-1.1.9-2 2-2s2 .9 2 2c0 2-3 1.75-3 5h2c0-2.25 3-2.5 3-5"
           + " 0-2.21-1.79-4-4-4z"),
+  INFO(
```

Comment:
**suggestion (blocking):** Please add an external link to where we can see the new icon if feasible

Reply:
Done!